shared/validate/schema/model/transaction/createPrepared/airdropReward.ts

```diff
@@ -0,0 +1,25 @@
+export const AIRDROP_REWARD = [
+    {
+        id: 'AIRDROP_REWARD',
+        type: 'object',
+        properties: {
+            sponsors: {
+                type: 'array',
+                items: {
+                    type: 'array',
+                    items: [
+                        {
+                            type: 'string',
+                            format: 'address'
+                        },
+                        {
+                            type: 'number'
+                        }
+                    ]
+                },
+                minItems: 0
```

Comment:
if possible to use constants here: maxItems should be `config.CONSTANTS.REFERRAL.MAX_COUNT`